python/vowpalwabbit/pyvw.py

```diff
@@ -597,11 +597,16 @@ def __init__(self, costs=[], prediction=0):
     def from_example(self, ex):
         class wclass:
             def __init__(self,
-                         label,
+                         action=None,
                          cost=0.,
                          partial_prediction=0.,
-                         probability=0.):
-                self.label = label
+                         probability=0.,
+                         **kwargs):
+                if kwargs.get('label', False):
+                    action = kwargs['label']
+                    warnings.warn("label has been deprecated. Please use 'action' instead.", DeprecationWarning)
+                self.label = action
```

Comment:
Do you know if there's any way to give a deprecation warning for an individual variable? We'll probably want to transition users away from calling object.label at some point.

If you don't know of any mechanism for it, its not a big deal though

Reply:
I am presently not familiar with any such way.